src/buffer/buffer.rs

```diff
@@ -381,6 +381,10 @@ impl fmt::Debug for Buffer {
     /// * `styles`: displayed as a list of: `{ x: 1, y: 2, fg: Color::Red, bg: Color::Blue,
     ///   modifier: Modifier::BOLD }` only showing a value when there is a change in style.
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        if self.area.is_empty() {
+            return f.write_fmt(format_args!("Buffer {{\n    area: {:?}\n}}", self.area));
```

Comment:
How about writing this on one line for compactness of representation?

Reply:
Then it's indentation differs from the one with content resulting in less easy visual comparison by the two.

Reply:
In what circumstances would this matter?

Reply:
`assert_eq!(Buffer::empty(…), actual)`

Reply:
In that case, `actual` is still displayed correctly which is all that matters, the test writing becomes change the Buffer::empty to Buffer::with_lines(<output copied from debug view of actual>), if actual is actually empty, then it doesn't really matter that much.